_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -368,13 +366,12 @@ public static function get_plans( $request ) {
 		 */
 		$use_cache = apply_filters( 'jetpack_cache_plans', true );
 
+		$data = $use_cache ? get_transient( 'jetpack_plans' ) : false;
+
 		if ( false === $data ) {
 			$path = '/plans';
-			$ip   = $_SERVER['HTTP_X_FORWARDED_FOR']; // used to calculate which currency to show
-			if ( empty( $ip ) ) {
-				// no ip list to forward, so create one:
-				$ip = $_SERVER['HTTP_CLIENT_IP'];
-			}
+			// passing along from client to help geolocate currency
```

Comment:
You probably want to start with `X_FORWARDED_FOR` in case there's a load balancer in front of the site (like cloudflare, AWS)